### DISCUSSION TITLE
Vertical scroll speed too fast (again)?

### DISCUSSION THREAD

**Discussion by enricotagliavini** [neutrinolabs/xrdp#2754] [Q&A]
Hello,

it looks like I'm experiencing the exact same issue as described in [#150](https://github.com/neutrinolabs/xorgxrdp/issues/150) , with a difference: it happens also when using the VNC backend, so it's not related to xorgxrdp.

The versions I'm using:
-  OS: CentOS Stream 8
- xrdp: 0.9.22.1-2
- xorgxrdp: 0.9.19-5

I tried with Remmina from Linux (Fedora) and Microsoft RDP client from MacOS and they both show the same issue. However from Microsoft Windows using RDP it works as expected, there is no scrolling issue from there. Using the same clients from Linux and MacOS, connecting to a Windows machine the scroll works fine, the issue with scrolling happens only when connecting to XRDP.

If I use a mouse with a scroll wheel the scrolling works as expected from all clients, same amount of scroll as on the client machine doing the same thing (e.g. scroll a web page or a folder with many files), but when using the two fingers scroll from a touchpad it does about 10 times the scrolling, making it next to impossible to use if you don't have access to a physical mouse, which is often the case with laptops.

I would appreciate very much any hint or advise you can share though, if any.

Iss there anything I can do to help investigating / fixing the issue?

**Comment by enricotagliavini** [comment]
One bit of information I forgot to include: using Remmina from Linux or Microsoft RDP from MacOS and connecting to a Windows machine doesn't show any issue with scrolling with the touchpad. The issue happens only when connecting to an XRDP server. Maybe it's an XRDP issue after all?

**Comment by metalefty** [comment]
Moved to https://github.com/neutrinolabs/xorgxrdp/issues/265.
